Documentation/PrivacyPolicy.md

```diff
@@ -1,35 +1,36 @@
 # Privacy Policy
 
-This policy applies to all information collected or submitted on the iPerf apps for iPhone and any other devices and platforms.
+This policy applies to all info collected or submitted from the iPerf apps for iPhone and any other devices and platforms.
 
-## Information we collect
+## Collected info
 
-We do not collect any data through the app, the only data that is exchanged is between the app and the iPerf 3 server you are connecting to. The help section is embedded in the app and does not connect to a server. We recommend [running your own iPerf 3 server](https://github.com/ndfred/iperf-ios/blob/master/Documentation/Help.md).
+No data is collected through the app. Data is only exchanged between the app and the iPerf 3 server it connects to. The help section is embedded in the app and does not connect to a server. It is recommended to [run your own iPerf 3 server](https://github.com/ndfred/iperf-ios/blob/master/Documentation/Help.md).
 
 When running a test:
 
-* the app connects to an iPerf 3 server and sends [the test's parameters](https://github.com/ndfred/iperf-ios/blob/master/Source/iperf3/iperf_api.c#L1526)
-* it then sends or receives random test data to / from the server
-* finally the [test results](https://github.com/ndfred/iperf-ios/blob/master/Source/iperf3/iperf_api.c#L1690) are sent to the server
+* The app connects to an iPerf 3 server and sends [the test's parameters](https://github.com/ndfred/iperf-ios/blob/master/Source/iperf3/iperf_api.c#L1526).
+* Random test data is sent to / and from the server.
+* Finally the [test results](https://github.com/ndfred/iperf-ios/blob/master/Source/iperf3/iperf_api.c#L1690) are sent to the server.
 
 ## Analytics
 
-The app does not collect any analytics beyond what is needed to run an iPerf 3 test as described in the prior section.
+The app itself does not collect any info, and its operation does not exchange any info with the server that is considered personal data as per [article 4 of the EU-GDPR](https://gdpr-info.eu/art-4-gdpr/). The app does not collect or send anything beyond what is described in "Collected info".
+The Apple App Store has its own privacy policy, which becomes effective if you get the app from there. Your Apple device also has its own privacy policy governance that is outside the realm of the iPerf app. See https://www.apple.com/privacy/control/
 
 ## Your Consent
 
-By using this app, you consent to our privacy policy.
+By using this app, you consent to exchange info as described above with the server you are using it with.
```

Comment:
Collecting consent this way is legally null and void. Explaining that it can't do anything if this doesnt happen, as that is what it does, is relevant.